images/linux/scripts/installers/oras-cli.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+################################################################################
+##  File:  oras-cli.sh
+##  Desc:  Installs ORAS Cli
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Determine latest ORAS CLI version
+ORAS_CLI_LATEST_VERSION_URL=https://api.github.com/repos/deislabs/oras/releases/latest
+ORAS_CLI_VERSION=$(curl $ORAS_CLI_LATEST_VERSION_URL | jq '.name' | tr -d '"' | cut -d 'v' -f 2)
+
+# Install ORAS CLI
+curl -LO https://github.com/deislabs/oras/releases/download/v${ORAS_CLI_VERSION}/oras_${ORAS_CLI_VERSION}_linux_amd64.tar.gz
+mkdir -p oras-install/
+tar -xzvf oras_${ORAS_CLI_VERSION}_linux_amd64.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_${ORAS_CLI_VERSION}_linux_amd64.tar.gz oras-install/
```

Comment:
let's move `oras_${ORAS_CLI_VERSION}_linux_amd64.tar.gz` to a variable to avoid duplication

Reply:
Thank you, moved `oras_${ORAS_CLI_VERSION}_linux_amd64.tar.gz` to a variable